eogrow/core/storage.py

```diff
@@ -18,15 +19,19 @@
 class StorageManager(EOGrowObject):
     PRESET_FOLDERS: Dict[str, str] = {"logs": "logs", "input_data": "input-data", "cache": "cache"}
 
-    class Schema(ManagerSchema):
+    class Schema(ManagerSchema, BaseSettings):
```

Comment:
so this inherits from both the `BaseModel` and `BaseSettings`... Does that cause any clashes? Does it still complain if a parameter is not set or an extra parameter is given? (I see that you extended the config, i'm just curious if its done correctly)

Reply:
As far as I can tell, everything works as it should and the config has been extended correctly. Looking at the implementations of these `pydantic` objects there are no implementation clashes.

As I mentioned, there is also an option that we change `EOGrowObject.Schema` to already inherit from `BaseSettings` and configure these extra parameters already there. This would in future allow us to have more such parameters out-of-the-box.

Reply:
I suggest we delay generalizing this to all configs until there is another usecase for it. But yes, this was something we already stumbled upon when researching how to pass secrets to eogrow configs.